<launch>
    <machine name="local" address="localhost" default="true"/>
    <node name="Chisel" pkg="chisel_ros" type="ChiselNode" output="screen"> 
        <param name="chunk_size_x" value="16"/>
        <param name="chunk_size_y" value="16"/>
        <param name="chunk_size_z" value="16"/>
        <param name="truncation_scale" value="10.0"/>
        <param name="integration_weight" value="1"/>
        <param name="use_voxel_carving" value="true"/>
        <param name="use_color" value="false"/>
        <param name="carving_dist_m" value="0.3"/>
        <param name="voxel_resolution_m" value="0.2"/>
        <param name="far_plane_dist" value="6"/>
        <param name="base_transform" value="world"/>
        <param name="depth_image_transform" value="/firefly/vi_sensor/camera_depth_optical_center_link"/>
        <param name="color_image_transform" value="sim_camera"/>
        <param name="fusion_mode" value="DepthImage"/>
    
        <remap from="/depth_image" to="/firefly/vi_sensor/camera_depth/depth/disparity"/>
        <remap from="/depth_camera_info" to="/firefly/vi_sensor/camera_depth/depth/camera_info"/>
        <remap from="/color_image" to="/sim_camera/color"/>
        <remap from="/color_camera_info" to="/sim_camera/camera_info"/>
        <!-- <remap from="/camera/depth_registered/points" to="/firefly/vi_sensor/camera_depth/depth/points"/> -->

    </node>
</launch>
